src/arns.lua

```diff
@@ -140,14 +189,36 @@ function arns.extendLease(from, name, years, currentTimestamp, msgId, fundFrom)
 	}
 end
 
+--- Calculates the extension fee for a given base fee, years, and demand factor
+--- @param baseFee number The base fee for the name
+--- @param years number The number of years
+--- @param demandFactor number The demand factor
+--- @return number The extension fee
 function arns.calculateExtensionFee(baseFee, years, demandFactor)
 	local extensionFee = arns.calculateAnnualRenewalFee(baseFee, years)
 	return math.floor(demandFactor * extensionFee)
 end
 
+--- @class IncreaseUndernameResponse
+--- @field record Record The updated record
+--- @field additionalUndernameCost number The additional undername cost
+--- @field baseRegistrationFee number The base registration fee
+--- @field remainingBalance number The remaining balance
+--- @field protocolBalance number The protocol balance
+--- @field df table The demand factor info
+--- @field fundingPlan table The funding plan
+--- @field fundingResult table The funding result
+
+--- Increases the undername limit for a record
+--- @param from string The address of the sender
+--- @param name string The name of the record
+--- @param qty number The quantity to increase the undername limit by
+--- @param currentTimestamp number The current timestamp
+--- @param msgId string The current message id
+--- @param fundFrom string|nil The intended payment sources; one of "any", "balance", or "stake". Default "balance"
+--- @return IncreaseUndernameResponse The response
```

Comment:
```suggestion
--- @return IncreaseUndernameResponse increaseUndenameResponse
```

Reply:
same in most this file, but not all are directly caused here. won't comment on the rest

Reply:
addressed a good chunk in [cebb3cb](https://github.com/ar-io/ar-io-network-process/pull/175/commits/cebb3cba577aa5a936939eb7bc99e01da74ebb9f)